pythnet/message_buffer/programs/message_buffer/src/instructions/put_all.rs

```diff
@@ -1,53 +1,46 @@
 use {
-    crate::{
-        state::*,
-        MessageBufferError,
-    },
+    crate::state::*,
     anchor_lang::prelude::*,
     std::mem,
 };
 
 
 pub fn put_all<'info>(
     ctx: Context<'_, '_, '_, 'info, PutAll<'info>>,
-    base_account_key: Pubkey,
+    _base_account_key: Pubkey,
     messages: Vec<Vec<u8>>,
```

Comment:
Why doesn't messages appear in the `#[instruction]` macro below?

Reply:
you can omit anything that won't be used. we need the base_account_key for the `seeds = []` but messages is only used in the actual logic